packages/embeds/embed-core/src/embed.ts

```diff
@@ -439,6 +439,10 @@ class CalApi {
     elementOrSelector: string | HTMLElement;
     config?: PrefillAndIframeAttrsConfig;
   }) {
+    if (this.cal.inlineEl) {
```

Comment:
Every namespace has his own cal instance, so we just check if the instance has inlineEl already defined.